garde/tests/rules/byte_length.rs

```diff
@@ -1,7 +1,10 @@
 use super::util;
+
+const UWU: usize = 101;
```

Comment:
pain

Reply:
HEY we need some tests and I'm not good with naming stuff